include/core/ib_particle.h

```diff
@@ -33,55 +33,62 @@
 using namespace dealii;
 
 /**
- * @brief This class defines values related to a particle used in the sharp interface IB. These values are used in the solver
+ * @brief Define the behavior of a specific particle for the sharp interface IB solver.
  *
- * @tparam dim  An integer that denotes the number of spatial dimensions
+ * Each particle defined will have these values used in the solver.
+ *
+ * @tparam dim An integer that denotes the dimension of the space in which
+ * the flow is solved.
```

Comment:
Is flow really the appropriate comment here for the tparam? Maybe we could not put the tparam at all or here you could say that dim is the number of spatial dimension.

Reply:
You are right, I changed it to the number of spatial dimensions.